mysql/0.0.1/main.tf

```diff
@@ -33,7 +31,12 @@ resource "helm_release" "mysql" {
 data "kubernetes_service" "mysql_service" {
   depends_on = [helm_release.mysql]
   metadata {
-    name      = "mysql-${random_pet.name_suffix.id}"
-    namespace = coalesce(var.namespace, random_pet.namespace.id)
+    name      = local.name
```

Comment:
Should we keep the `mysql-` prefix?

Reply:
The resource deployed by this chat will include mysql suffix